src/test/java/com/znsio/sample/e2e/screen/web/amazon/ShoppingCartScreenWeb.java

```diff
@@ -0,0 +1,77 @@
+package com.znsio.sample.e2e.screen.web.amazon;
+
+import com.context.TestExecutionContext;
+import com.znsio.e2e.runner.Runner;
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.screen.amazon.ShoppingCartScreen;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+
+public class ShoppingCartScreenWeb extends ShoppingCartScreen {
+
+    private static final String SCREEN_NAME = ShoppingCartScreenWeb.class.getSimpleName();
+    private static final Logger LOGGER = Logger.getLogger(SCREEN_NAME);
+
+    private final Driver driver;
+    private final TestExecutionContext context;
+    private final Visual visually;
+
+    private static final By cartButton = By.id("nav-cart");
+
+    private static final By shoppingCartHeader=By.xpath("//div[@id='sc-active-cart']//child::h1");
+    private static final By productTitle= By.xpath("//div[@class='sc-list-item-content']//span[@class='a-truncate-cut']");
+    private static final By productPrice = By.xpath("//div[@class='sc-list-item-content']//span[@class='currencyINR']/parent::span");
+    private static final By productCount = By.className("a-dropdown-prompt");
+    public ShoppingCartScreenWeb(Driver driver, Visual visually) {
+        long threadId = Thread.currentThread().getId();
+        this.driver = driver;
+        this.visually = visually;
+        this.context = Runner.getTestExecutionContext(threadId);
+    }
+
+    @Override
+    public boolean isUserOnShoppingCartPage() {
+        LOGGER.debug("On Shopping Cart Page");
+        visually.checkWindow(SCREEN_NAME,"On Shopping Cart page");
+        return driver.isElementPresent(shoppingCartHeader);
+    }
+
+    @Override
+    public String getProductTitle() {
+        String productTitleText = driver.findElement(productTitle).getText().trim();
+        LOGGER.info("Product Title is: " + productTitleText);
+        return productTitleText;
+    }
+
+    @Override
+    public ShoppingCartScreen moveToShoppingCartPage() {
+        LOGGER.info("Click on Amazon Cart Button ");
+        driver.findElement(cartButton).click();
+        return this;
+    }
+
+    @Override
+    public String getProductCount() {
+        String productCountText = driver.findElement(productCount).getText().trim();
+        LOGGER.info("Product Quantity is: "+ productCountText);
+        return productCountText;
+    }
+
+    @Override
+    public String getProductPrice() {
+        String productPriceText=driver.findElement(productPrice).getText().trim();
+        LOGGER.info("Product Price is: "+ productPriceText);
+        return productPriceText;
+    }
+
+    @Override
+    public boolean verifyAddedProductInCart(String addedProductName) {
+            if(getProductTitle().equals(addedProductName) && getProductCount().equals(1)){
+                return true;
+            }
+            else
+                return false;
+    }
+
```

Comment:
this method can be rewritten as 

public boolean verifyAddedProductInCart(String addedProductName) {
            return (getProductTitle().equals(addedProductName) && getProductCount().equals(1));     
    }

Reply:
add logs too